packages/orchestration/test/examples/send-anywhere.test.ts

```diff
@@ -253,3 +251,189 @@ test('baggage', async t => {
   const tree = inspectMapStore(contractBaggage);
   t.snapshot(tree, 'contract baggage after start');
 });
+
+test('failed ibc transfer returns give', async t => {
+  t.log('bootstrap, orchestration core-eval');
+  const {
+    bootstrap,
+    commonPrivateArgs,
+    brands: { ist },
+    utils: { inspectLocalBridge, pourPayment, inspectBankBridge },
+  } = await commonSetup(t);
+  const vt = bootstrap.vowTools;
+
+  const { zoe, bundleAndInstall } = await setUpZoeForTest();
+
+  t.log('contract coreEval', contractName);
+
+  const installation: Installation<StartFn> =
+    await bundleAndInstall(contractFile);
+
+  const storageNode = await E(bootstrap.storage.rootNode).makeChildNode(
+    contractName,
+  );
+  const sendKit = await E(zoe).startInstance(
+    installation,
+    { Stable: ist.issuer },
+    {},
+    { ...commonPrivateArgs, storageNode },
+  );
+
+  t.log('client sends an ibc transfer we expect will timeout');
+
+  const publicFacet = await E(zoe).getPublicFacet(sendKit.instance);
+  const inv = E(publicFacet).makeSendInvitation();
+  const amt = await E(zoe).getInvitationDetails(inv);
+  t.is(amt.description, 'send');
+
+  const anAmt = ist.make(SIMULATED_ERRORS.TIMEOUT);
```

Comment:
That's distracting. There's nothing error-ish about this use, AFAICT.  I'd prefer
```suggestion
  const arbitraryNat = 37n;
  const anAmt = ist.make(arbitraryNat);
```

Reply:
We've been using the `SIMULATED_ERRORS` constant to make it clear we are intentionally throwing an error in the test:
https://github.com/Agoric/agoric-sdk/blob/66e4cc0f36b5ed33f1b3fa235a335757ca868b9c/packages/vats/tools/fake-bridge.js#L171-L180

The IBC bridge (used here) isn't trained to throw on these magic constants - it throws if it receives a message that doesn't have a handler registered via `addMockAck`. But the `Localchain` bridge is trained to do this, so I think consistently adopting this practice makes intentional errors clearer.

Reply:
>  But the Localchain bridge is trained to do this,

It throws when it sees Amounts with these magic values?  It can be difficult to inject errors, so I'll go along with this.

Reply:
Yes, see `fakeLocalChainBridgeTxMsgHandler`: https://github.com/Agoric/agoric-sdk/blob/c1c5a6fd2f347b96d1e514bd0bf98550639e349f/packages/vats/tools/fake-bridge.js#L193-L228

> It can be difficult to inject errors

+1